java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_outer_classname = "IncompleteExample";
+
+message Example {
+    string ytid = 1;
```

Comment:
ytid sounds obscure. video_id or something like that?

Reply:
Consider adding comments to all the fields you have in a proto in addition to having good field names that are a bit more descriptive.

Reply:
Oh. I meant it as YouTube Video Id. I can change it to video_id and add comments as well.